runtime/Runtest/intrinsics.c

```diff
@@ -69,8 +69,9 @@ void recursively_allocate(KTestObject *obj, size_t index, void *addr,
                            obj->pointers[i].index, 0, 1);
     }
     void *offset_addr = (void *)(addresses[index] + (obj->pointers[i].offset));
-    memcpy(offset_addr, &addresses[obj->pointers[i].index], sizeof(void *));
```

Comment:
Likely it is okay. Just to be sure: was it a bug?

Reply:
Yes, it was.